SwrveSDK/src/main/java/com/swrve/sdk/config/SwrveConfigBase.java

```diff
@@ -16,6 +18,20 @@
  * Configuration for the Swrve SDK.
  */
 public abstract class SwrveConfigBase {
+
+    /**
+     * Default Log Tag
+     */
+
+    private final String LOG_TAG = "SwrveConfigBase";
+
+    /**
+     * List of supported stacks
+     */
+    public enum SwrveStack{
+        EU, US, ACTIVISION, FEATURESTACK
```

Comment:
I saw the comments but in the interest of convenience in our SDK, I thought I'd add them in.
OK if featurestacks aren't allowed and thats final. I'll remove them.
I definitely think activision shouldn't be in that case. We have a single customer who uses it, not multiple customers.

As for your last comment, I'm not sure what you mean. Do you mean make the constructor accept a stack paramater?